docs/Releases.md

```diff
@@ -1,17 +1,19 @@
 # Releases
 
 Since Homebrew 1.0.0 most Homebrew users (those who haven't run a `dev-cmd` or
-set `HOMEBREW_DEVELOPER=1`) require tags on the [Homebrew/brew repository](https://github.com/homebrew/brew)
+set `HOMEBREW_DEVELOPER=1` which is ~99.9% based on analytics data) require tags
+on the [Homebrew/brew repository](https://github.com/homebrew/brew)
 in order to get new versions of Homebrew. There are a few steps in making a new
 Homebrew release:
 
 1. Check the [Homebrew/brew pull requests](https://github.com/homebrew/brew/pulls),
    [issues](https://github.com/homebrew/brew/issues),
-   [Homebrew/core issues](https://github.com/homebrew/homebrew-core/issues) and
+   [Homebrew/homebrew-core issues](https://github.com/homebrew/homebrew-core/issues) and
    [Homebrew/discussions (forum)](https://github.com/homebrew/discussions/discussions) to see if there is
    anything pressing that needs to be fixed or merged before the next release.
    If so, fix and merge these changes.
-2. Ensure that no code changes have happened for at least a couple of hours (ideally 24 hours)
+2. Ensure that no code changes have happened for at least a couple of hours (ideally 4 hours),
+   at least one Homebrew/homebrew-core pull request CI job has completed successfully,
```

Comment:
```suggestion
2. Ensure that no code changes have happened for at least a couple of hours (ideally 4 hours) and
   at least one Homebrew/homebrew-core pull request CI job has completed successfully,
```

Do we want to also mention that Homebrew/brew CI should have passed on the latest commit?

Reply:
I would say ideally yes but given how flaky it is it's not something I've checked. It could be rerun in that case, though? How reflective do you think Homebrew/brew CI status is of user issues?

Reply:
I just scrolled back through the commit history and it looks like the only cases where CI fails is on code-coverage failures. I have seen other CI errors appear on `master`, but I think it would be fair to say that they tend to be minor issues that don't really affect users (e.g. `brew man` failing).

I think there is potential for it to catch more significant errors, though. If two PRs modify separate parts of the `install` code and aren't tested with each other, I could see that causing `brew tests` to pass in each individual PR while failing once the changes are combined on `master`.

That's a more extreme example, though. I, personally, think it doesn't hurt to add it. Since CI is almost always green, it does seem like when it is red it deserves some attention. You're right, though, it's not as good of an indication of the state of the code from a user standpoint, so I'm fine if it's not added.

Reply:
> That's a more extreme example, though. I, personally, think it doesn't hurt to add it. Since CI is almost always green, it does seem like when it is red it deserves some attention. You're right, though, it's not as good of an indication of the state of the code from a user standpoint, so I'm fine if it's not added.

I could at least add "check the state of CI on `master`" or similar. Thanks for the idea!